### PR TITLE
create table test performed table

### PR DESCRIPTION
**Pull request by BobanL**
# BACKEND PULL REQUEST

## Related Issue

- #4940 

## Changes Proposed

- Add a new table to store the test performed code

## Additional Information

- A new entity table will be used to store any information between the device type and supported disease

### CODE REVIEW

File: backend/src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -4333,3 +4333,45 @@ databaseChangeLog:
...
                   type: text
+  - changeSet:
+      id: create-test-performed-table
+      author: boban@skylight.digital
+      comment: creates a table to store the test performed code between device type and supported disease
+      changes:
+        - tagDatabase:
+            tag: create-test-performed-table
+        - createTable:
+            tableName: test_performed
```
Comment (emmastephenson) [inline backend/src/main/resources/db/changelog/db.changelog-master.yaml:4344]: nit: I would name this something more descriptive like `test_performed_loinc_code`

**Review by emmastephenson — DISMISSED** [review]
Looks good!

File: backend/src/main/resources/db/changelog/db.changelog-master.yaml [hunk 2]
```diff
@@ -4333,3 +4333,45 @@ databaseChangeLog:
...
+                  remarks: The reference to the supported disease.
+                  type: uuid
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk__device_supported_disease__supported_disease_type
+                    references: supported_disease
+              - column:
+                  name: test_performed_code
+                  type: text
+                  remarks: The code that identifies which disease was tested.
```
Comment (zdeveloper) [inline backend/src/main/resources/db/changelog/db.changelog-master.yaml:4372]: this should also have `nullable: false` constraint

File: backend/src/main/resources/db/changelog/db.changelog-master.yaml (hunk 1 above)
Comment (zdeveloper) [reply backend/src/main/resources/db/changelog/db.changelog-master.yaml:4344]: `device_test_performed_loinc_code`?

**Review by emmastephenson — APPROVED** [review]
LGTM!

**Review by zdeveloper — DISMISSED** [review]

**Review by zdeveloper — APPROVED** [review]

**Review by emmastephenson — APPROVED** [review]

**Pull request merged**